src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -565,6 +566,95 @@ describe('LibraryServiceClient', function() {
     });
   });
 
+  describe('streamShelves', function() {
+    it('invokes streamShelves without error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var request = {};
+
+      // Mock response
+      var expectedResponse = {};
+
+      // Mock Grpc layer
+      client._streamShelves = function() {
+        return new MockStream(request, expectedResponse);
+      };
+
+      var stream = client.streamShelves().on('data', function(response) {
```

Comment:
Thinking of this in the perspective of tests serving as a sort of documentation, it seems a bit strange to have a test for a client streaming method to have a listener for responses.